node/src/runtime/mainnet.rs

```diff
@@ -114,17 +151,21 @@ pub fn testnet_genesis(
 		},
 		sudo: SudoConfig {
 			// Assign network admin rights.
-			key: Some(sudo_key),
+			key: Some(multisig.clone()),
 		},
 
 		// Monetary
 		balances: BalancesConfig {
-			// Configure endowed accounts with initial balance of 1 << 60.
-			balances: endowed_accounts
-				.iter()
-				.cloned()
-				.map(|(k, endowment)| (k, endowment as u128 * GGX))
-				.collect(),
+			balances: [
+				(
+					multisig.clone(),
+					TOTAL_SUPPLY - (1100 * GGX) - (500 * GGX * multisig_owners.len() as u128),
```

Comment:
@akorchyn So do we keep 1000 tokens for min staking?

Reply:
Yeah :( it requires to careful changes in code